assets/l10n/intl_ar.arb

```diff
@@ -637,7 +637,7 @@
             "link": {}
         }
     },
-    "isTyping": "يكتب…",
+    "isTyping": "يكتب",
```

Comment:
so interesting, what is it? why you change like that?

Reply:
you want to put the text stable and animate the flashingCircle?

Reply:
I removed every "..." because these strings as used in chat list item (with ...) and in app bar (with flashing circle) so it's easier to manage like so